src/core/addonManager.js

```diff
@@ -1,13 +1,10 @@
 /* global window */
-
 import log from 'core/logger';
 import {
+  installEventList,
```

Comment:
Seems like this should be `INSTALL_EVENT_LIST` if it's a constant, even though it seems this is from a former commit and not introduced here.